includes/Installer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+namespace WeDevs\WePOS;
+
+if ( ! defined( 'ABSPATH' ) ) {
+    exit;
+}
+
+/**
+ * Installer class.
+ */
+class Installer {
+
+    /**
+     * Run installer.
+     *
+     * @since WEPOS_LITE_SINCE
+     *
+     * @return void
+     */
+    public function run() {
+        $this->add_installation_data();
+        $this->create_tables();
+    }
+
+    /**
+     * Add installation data.
+     *
+     * @since WEPOS_LITE_SINCE
+     *
+     * @return void
+     */
+    public function add_installation_data() {
+        $installed = get_option( 'we_pos_installed' );
+
+        if ( ! $installed ) {
+            update_option( 'we_pos_installed', time() );
+        }
+
+        if ( function_exists( 'dokan' ) ) {
+            $users_query = new \WP_User_Query( [
+                'role__in' => [ 'seller', 'vendor_staff' ]
+            ] );
+            $users       = $users_query->get_results();
+
+            if ( count( $users ) > 0 ) {
+                foreach ( $users as $user ) {
+                    $user->add_cap( 'publish_shop_orders' );
+                    $user->add_cap( 'list_users' );
+                }
+            }
+        }
+
+        update_option( 'we_pos_version', WEPOS_VERSION );
+        set_transient( 'wepos-flush-rewrites', 1 );
+    }
+
+    /**
+     * Create database tables.
+     *
+     * @since WEPOS_LITE_SINCE
+     *
+     * @return void
+     */
+    public function create_tables() {
+        global $wpdb;
+
+        include_once ABSPATH . 'wp-admin/includes/upgrade.php';
+
+        $tables = [
+            "CREATE TABLE IF NOT EXISTS `{$wpdb->prefix}wepos_product_logs` (
+                `id` bigint unsigned NOT NULL AUTO_INCREMENT,
+                `product_id` bigint unsigned NOT NULL,
+                `product_title` varchar(255) NOT NULL,
+                `product_type` varchar(255) NOT NULL,
```

Comment:
Why does `product_type` need `255` length? I guess it can be covered within `100`.
Same concern for `product_sku`.
Kindly recheck these and try to optimize.